docs/book/src/quick-start.md

```diff
@@ -7,6 +7,29 @@ This Quick Start guide will cover:
 - [Running locally](#test-it-out)
 - [Running in-cluster](#run-it-on-the-cluster)
 
+# Overview
```

Comment:
It is not a content for the quick start.
The goal of the quick start is only provide the instructions for a quick-start.
We should not have very verbose instructions.

See the comment: https://github.com/kubernetes-sigs/kubebuilder/issues/3196#issuecomment-1415494809

The idea is to create a **new** doc section called overview which would be after the quick-start section ( look at the left side of the menu )

Then, in this section we can add the content. 
Also, we should NOT link https://dev4devs.com/2020/08/16/how-to-getting-started-develop-go-operators-from-scratch-with-sdk-1-0/. The idea is to add to this new section a similar content. See that the content/syntax used in the blog is outdated. You can copy and paste and the update / improve it.

Also, in this section we should add the other points add to the comment https://github.com/kubernetes-sigs/kubebuilder/issues/3196#issuecomment-1415494809. We can also create more one new section called "Common Suggestions" or "Good Practices" and add those items there. Afterwords just link and suggest users check it out after read the overview. 

I hope that can help out. 

/hold

Reply:
Thanks for clarifying. I will surely work on that.

Reply:
About the dev4blog I think there is much info to process in this blog. There are steps tutorial so I dont think I cantake things out of it and add it on blog. WDYT?

Reply:
About the dev4 blog. The blog is quite of old. it is a tutorial for making a operator sdk and we already have crojob tutorial docs for that. If you want to give blog reference in docs. I have found one https://yash-kukreja-98.medium.com/develop-on-kubernetes-series-operator-dev-the-introduction-1dcbaadc9fa7 WDYT?

Reply:
Hi @Sajiyah-Salat,

That still not exactly what s suggested in the comment https://github.com/kubernetes-sigs/kubebuilder/issues/3196#issuecomment-1415494809. The idea is mainly

- New doc with the name "Good Practices" and add what you added here. 
- Then, create a new doc, "Overview", which would have similar content https://dev4devs.com/2020/08/16/how-to-getting-started-develop-go-operators-from-scratch-with-sdk-1-0/. You'll need to read, follow the steps and update it according to kubebuilder latest scaffold. Also, it will be required some tweaks and the other links as reference as well. The goal is NOT to add the link or blog references. The idea is to provide a very small and fast overview to make easier the new users' skills up faster in the basics. 
-

Reply:
dev4 blog teach us to make a operator. However we already have docs about that. what do you want me to extract from the blog then.

Reply:
dev4 blog teach us to make a operator. However we already have docs about that. what do you want me to extract from the blog then.

Reply:
Our tutorials are quite extensive. The aim was to develop a concise overview of the blog, allowing users to grasp the general workings without having to delve deeply into the entire tutorial series. Currently, we lack this condensed document, and as a result, when assisting community members, I typically share the blog for their reference.